main/webapp/modules/core/styles/index/open-project-ui.css

```diff
@@ -220,3 +220,57 @@ ul#tagsUl li a {
   background: #28458A;
   color: white;
 }
+
+/* Search bar in open project */
+/* search */
+#divSearch{
+   float: left;
+   width: 2.5%;
+   margin-left: -0.5%;
+}
+
+#search-icon {
+  float: left;
+  width: 22px;
+}
+
+#divInput {
```

Comment:
This strikes me as a too unspecific DOM id.

Reply:
@wetneb: Do you mean changing the ID from `divInput` to `searchInput`?

Reply:
Yes that would be a better name

Reply:
Done.